vendor/vendor.json

```diff
@@ -447,31 +384,28 @@
 			"versionExact": "v1.5.4"
 		},
 		{
-			"checksumSHA1": "+RHv6D0Db8Yke6G9cxGjEW61viM=",
-			"origin": "github.com/cockroachdb/cockroach/vendor/github.com/gogo/protobuf/jsonpb",
```

Comment:
Do you have the latest govendor? Why did the origin field disappear?

Reply:
> Do you have the latest govendor?

Yes.

> Why did the origin field disappear?

Because I'm now using the canonical origin, which is the same as `path`. See commit description for details.